test/JDBC/jdbc_schedule

```diff
@@ -7,9 +7,7 @@
 # 5. To add a test, add test name (without extension Eg. BABEL-TEST) on a
```

Comment:
Fix: remove this file change since you're not adding anything new in the schedule file.